### PR TITLE
Update the acmcsuf.com/teams page with the Fall 2024 Board

### LINKED ISSUE #1103: Marketing logo interference with marketing board face images
### What happened?

marketing_issue

The marketing logo looks like it moves right on top of my face image making it look a bit too close to my picture, compared to other team logos which appear more distant to the face images they are close to.

### PR DESCRIPTION
**Pull request by 26samaahmed**
Add/Edit Board Info for Fall 2024

Screenshot 2024-06-28 210110
Screenshot 2024-06-28 210119
Screenshot 2024-06-28 210128
Screenshot 2024-06-28 210137
Screenshot 2024-06-28 210218
Screenshot 2024-06-28 210227
Screenshot 2024-06-28 210237

### CODE REVIEW

File: src/routes/(site)/teams/officer-profile.svelte [hunk 1]
```diff
@@ -31,6 +32,7 @@
   <div>
     <h3 class="acm-heavier">{officerName}</h3>
     <p class="acm-heavy">{@html titleHTML}</p>
+    <p>@{officerDiscord}</p>
```
Comment (EthanThatOneKid) [inline src/routes/(site)/teams/officer-profile.svelte]: Copy the Discord username when clicked. Change the style on hover. 

- Prior art: <https://github.com/EthanThatOneKid/acmcsuf.com/blob/134d89cef204ae3d61a5f2494152139ebc63443c/src/routes/(site)/colors/color.svelte#L17>

**Review by EthanThatOneKid — COMMENTED** [review]
I'm aware not _all_ of the officer data is up to date for Fall 2024 just yet, but so far so good! Looking great, @26samaahmed 😎

File: src/routes/(site)/teams/officer-profile.svelte (hunk 1 above)
Comment (EthanThatOneKid) [reply src/routes/(site)/teams/officer-profile.svelte]: Sweet! Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1099/commits/55b173bc7dc5084613ef5715faa825fdeac47efd>.

**Comment by 26samaahmed** [comment]
All board members have been added along with their discord usernames. The Nodebuds and Marketing Team animation were also fixed.

https://github.com/user-attachments/assets/892ed322-c18b-4852-a641-67b7970081b1

File: src/routes/(site)/teams/+page.svelte [hunk 2]
```diff
@@ -32,6 +32,7 @@
         gateways for students to explore new fields, develop new interests, and learn new skills
         that will benefit them in the industry.
       </p>
+      <p><span style="color:red">*</span>Feel free to reach out to board members through their discord, stated below their profile.</p>
```
Comment (EthanThatOneKid) [inline src/routes/(site)/teams/+page.svelte]:
```suggestion
      <p>Feel free to reach out to board members through their Discord username, stated below their profile.</p>
```

Comment (EthanThatOneKid) [reply src/routes/(site)/teams/+page.svelte]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1099/commits/044a292ec9dac6d99f9e07edbed81caf12424f52>.

**Comment by 26samaahmed** [comment]
Edited Nodebuds Description + moved Marketing team to below the Game Dev Team to maintain alphabetical order.

https://github.com/user-attachments/assets/91230f31-4793-4893-8da4-43c712e1252a

**Review by EthanThatOneKid — APPROVED** [review]
This is awesome, congrats on merging your first major website PR officially as webmaster! Great work, @26samaahmed! 🎉

**Pull request merged**